src/internal/polyfills/Array.from.js

```diff
@@ -0,0 +1,24 @@
+import isIterable from '../../isIterable';
+import isUndefined from '../../isUndefined';
+
+const copyArray = (items, mapFn, thisArg) => {
+  if (isUndefined(mapFn)) return [...items];
+  if (isUndefined(thisArg)) return [...items].map(mapFn);
+  return [...items].map(mapFn).bind(thisArg);
+};
+
+const fromArray = (items, mapFn = undefined, thisArg = undefined) => {
```

Comment:
The reason to set the defaults is imitate the behavior of `Array.from` when using `R.curry` for example:

`R.curry(Array.from)`
`> ƒ r(e){return 0===arguments.length||n(e)?r:t.apply(this,arguments)}`

Reply:
@rafaelpadovezi if you curry the function which has default param values you need to explicitly pass the `undefined` value to trigger usage of default param value. In our case `undefined` is default value so that's why I'm saying it doesn't make sense to define them explicitly.

more on the subject of currying and default params in this article: https://www.linkedin.com/pulse/function-currying-default-params-javascript-vladim%C3%ADr-gorej/

Reply:
Thanks for the clarification @char0n. I removed the default values and fixed the other comments.